### PR TITLE
Final fixes for solr improvements.

### PR DESCRIPTION
**Pull request by almathew**
## What's this PR do?
Makes the final necessary fixes of issues discovered during load testing of new code.

### CODE REVIEW

**Comment by koonsies** [comment]
Any tips on how to go about testing this? I'm not too familiar with our solr setup, so I may not be very valuable as a reviewer here.

**Comment by almathew** [comment]
Pull the latest change, and I'll go ahead and walk you through it on Slack.

File: seo/views/search_views.py [hunk 1]
```diff
@@ -1274,16 +1279,26 @@ def company_listing(request, alpha=None, group=None):
...
 def solr_ac(request):
-    """Populate the searchbox autocomplete."""
+    """
+    Populate the searchbox autocomplete.
 
+    """
+    callback = request.GET.get('callback')
     lookup_type = request.GET.get('lookup')
     term = request.GET.get('term')
+    # Autocomplete shouldn't trigger until at least 3 characters have been
```
Comment (vigowei) [inline seo/views/search_views.py]: 2 or 3? code says 2
Front-end should not send any request before minimum characters are typed, so we don't waste cycles returning []

Comment (almathew) [inline seo/views/search_views.py]: 2. I thought it was 3, but then demoing it for @mwarnsleyDE made me realize it was 2. Fixed.

**Pull request merged**